TestAssets/TestProjects/TestAppWithLibrary/TestLibrary/project.json

```diff
@@ -10,9 +10,9 @@
     ]
   },
   "dependencies": {
-    "NETStandard.Library": "1.5.0-rc2-24027"
+    "NETStandard.Library": "1.5.0-rc3-24113-00"
   },
   "frameworks": {
-    "netstandard1.5": {}
+    "netstandard1.6": {}
```

Comment:
That's expected.  NETStandard1.X is not meant to be a runnable TFM.  The reason you are hitting this is because you are resolving runtimes for a NETStandard TFM.  Instead use NETCoreApp1.0 if you want something that's guaranteed to be run-able.